common/src/main/java/com/sequenceiq/cloudbreak/telemetry/DataBusEndpointProvider.java

```diff
@@ -11,6 +14,11 @@
 
     private static final String CNAME_ENDPOINT = "https://dbusapi.us-west-1.altus.cloudera.com";
 
+    private static final Map<String, String> DBUS_API_S3_ENDPOINT_PATTERN_PAIRS =
```

Comment:
I think this should be a config property

Reply:
do we have any common config that can be applied for many of our services? (like a common application.yml) because it would make sense here, if there is no I can create one

Reply:
we dont have I think

Reply:
@doktoric , i have eliminated the duplications (abstract class -> common, abstract CM based class -> core-api), regarding the common configuration change, maybe that is out of the context of this change as that would be useful for other static configs as well, if you are ok with it, I would do that in ta different PR

Reply:
resolving this as CB-11658 will cover that